backend/views/syson-tree-explorer-view/src/main/java/org/eclipse/syson/tree/explorer/view/services/SysONDefaultExplorerServices.java

```diff
@@ -57,13 +58,16 @@ public class SysONDefaultExplorerServices implements ISysONDefaultExplorerServic
 
     private final UtilService utilService = new UtilService();
 
+    private final ExplorerServices siriuswebExplorerService;
+
     public SysONDefaultExplorerServices(IIdentityService identityService, IContentService contentService, IRepresentationMetadataSearchService representationMetadataSearchService, IExplorerServices explorerServices,
-            ISysONExplorerFilterService filterService) {
+            ISysONExplorerFilterService filterService, ExplorerServices siriuswebExplorerService) {
```

Comment:
You don't need `ExplorerServices` from Sirius Web.
There is already a `IExplorerServices explorerServices` variable that gives you access to `hasChildren` method.
We don't have any implementation of `IExplorerServices` in SysON so the only implementation returned by `IExplorerServices explorerServices` is the one coming from Sirius Web.